features/support/env.rb

```diff
@@ -70,6 +70,7 @@
 # recommended as it will mask a lot of errors for you!
 #
 ActionController::Base.allow_rescue = false
+$arel_silence_type_casting_deprecation=true
 
```

Comment:
Short term fix for arel deprecation warnings, will need to be revisited when creating the next bump.